lib/classes/InventoryFolder.ts

```diff
@@ -146,6 +164,32 @@ export class InventoryFolder
         throw new Error('Failed to create inventory folder');
     }
 
+    async delete(saveCache: boolean = false): Promise<void> {
+        const { caps } = this.agent.currentRegion;
+        const invCap = await caps.getCapability('InventoryAPIv3');
+
+        await this.agent.currentRegion.caps.requestDelete(`${invCap}/category/${this.folderID}`)
+        const ids = this.getChildFoldersRecursive().map(x => x.folderID)

```

Comment:
This map adds an additional iteration of the whole set, let's just call .folderID directly below